src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.spec.ts

```diff
@@ -6,9 +6,11 @@ import { Component, NO_ERRORS_SCHEMA } from '@angular/core';
 import { ReactiveFormsModule } from '@angular/forms';
 import { ActivatedRoute, Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
-import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
+import { ExuiCommonLibModule, FilterService } from '@hmcts/rpx-xui-common-lib';
 import { staffFilterOptionsTestData } from '../../../test-data/staff-filter-options.test.data';
 import { StaffAddEditUserFormComponent } from './staff-add-edit-user-form.component';
+import { of } from 'rxjs';
+import { StaffUserCheckAnswersComponent } from '../staff-user-check-answers/staff-user-check-answers.component';
```

Comment:
Imports are listed incorrectly here, 'rxjs' should be above all indirect imports, staffaddedituserformcomponent should be the last import listed

Reply:
@connorpgpmcelroy ,  `lint` should highlight it. 

`ng lint --fix`    - should resolve all these minor issues.